src/cheri_insts.sail

```diff
@@ -229,6 +230,46 @@ function clause execute (CGetOffset(rd, cs1)) = {
   RETIRE_SUCCESS
 }
 
+/*!
+ * Integer register *rd* is set equal to the **upper half** of capability
+ * register *cs*.
+ *
+ * The bits returned here are of the **in-memory** form of the capability, which
+ * differs from the compressed in-register form by the XOR of [null_cap_bits].
```

Comment:
I'm not sure if we should be calling this the `in-register` form since I believe all/most implementations actually use a different representation for the registers. That being said, I don't have a better naming suggestion, so I guess we can keep it :)

Reply:
Indeed, what's in a register is an implementation detail, so I don't like mentioning this here. I'd just delete it entirely.